apps/src/templates/rubrics/RubricSettings.jsx

```diff
@@ -299,69 +172,44 @@ export default function RubricSettings({
+        setStatusAll(STATUS_ALL.ERROR);
       }
     });
   };
 
   return (
     <div
-      className={classnames(
-        'uitest-rubric-settings',
-        {[style.settings]: !experiments.isEnabled('ai-rubrics-redesign')},
-        {
-          [style.settingsVisible]: visible,
-          [style.settingsHidden]: !visible,
-        }
-      )}
+      className={classnames('uitest-rubric-settings', {
+        [style.settingsVisible]: visible,
+        [style.settingsHidden]: !visible,
+      })}
     >
-      {!experiments.isEnabled('ai-rubrics-redesign') && (
-        <Heading2>{i18n.settings()}</Heading2>
-      )}
-      {experiments.isEnabled('ai-rubrics-redesign') && (
-        <div className={style.studentInfoGroup}>
-          <Heading5>
-            {i18n.lessonNumbered({
-              lessonNumber: lesson.position,
-              lessonName: lesson.name,
-            })}
-          </Heading5>
-          <div className={style.selectors}>
-            <SectionSelector reloadOnChange={true} requireSelection={false} />
-          </div>
+      <div className={style.studentInfoGroup}>
+        <Heading5>
+          {i18n.lessonNumbered({
+            lessonNumber: lesson.position,
+            lessonName: lesson.name,
+          })}
+        </Heading5>
+        <div className={style.selectors}>
+          <SectionSelector reloadOnChange={true} requireSelection={false} />
         </div>
-      )}
-
-      {canProvideFeedback && !experiments.isEnabled('ai-rubrics-redesign') && (
-        <div className={style.aiAssessmentOptions}>
-          <div>
-            <BodyTwoText>
-              <StrongText>{i18n.aiAssessment()}</StrongText>
-            </BodyTwoText>
-            <BodyTwoText>{i18n.runAiAssessmentDescription()}</BodyTwoText>
-          </div>
-          <Button
-            className="uitest-run-ai-assessment"
-            text={studentButtonText()}
-            color={Button.ButtonColor.brandSecondaryDefault}
-            onClick={handleRunAiAssessment}
-            style={{margin: 0}}
-            disabled={status !== STATUS.READY}
-          >
-            {polling && <i className="fa fa-spinner fa-spin" />}
-          </Button>
-          <BodyTwoText className="uitest-eval-status-text">
-            {statusText() || ''}
-          </BodyTwoText>
-          <div>
-            <BodyTwoText>
-              <StrongText>{i18n.aiAssessmentAll()}</StrongText>
+      </div>
+
+      <div className={style.settingsGroup}>
+        <Heading2>{i18n.aiAssessment()}</Heading2>
+        <div className={style.settingsContainers}>
+          <div className={style.runAiAllStatuses}>
+            <BodyTwoText className="uitest-eval-status-all-text">
+              <StrongText>{summaryText()}</StrongText>
             </BodyTwoText>
-            <BodyTwoText>{i18n.runAiAssessmentDescriptionAll()}</BodyTwoText>
+            {statusAllText() && (
+              <BodyTwoText>{statusAllText() || ''}</BodyTwoText>
```

Comment:
Can you explain why the `|| ''` is needed?  it seems as if `statusAllText()` returns null this `BodyTwoText` wouldn't show up and if if `statusAllText()` returns `''` that would be printed here anyway...